cluster/config-defaults.yaml

```diff
@@ -65,6 +65,9 @@ skipper_liveness_init_delay_seconds: 30
 skipper_readiness_init_delay_seconds: 1
 skipper_liveness_init_delay_seconds: 30
 {{end}}
+# skipper termination settings
+skipper_terminationGracePeriod: "70s"
```

Comment:
`termination_grace_period`, to match the rest of the config items?

Reply:
Also, it should be a number in seconds. I'm not sure if we have a template function to convert, but I'm not sure if it's even needed.

Reply:
It needs to be:
```
timeouts < skipper_wait_for_healthcheck_interval < termination_grace_period 
```

Reply:
Ah sorry I can't read :) 
Of course: `skipper_termination_grace_period` :)